Style-guide/README.md

```diff
@@ -81,7 +78,7 @@ Descriptive and consistent naming makes software easier to read and understand.
 - naming methods for their side effects
   - verb methods follow the -ed, -ing rule for the non-mutating version
```

Comment:
I think this needs clarification and examples.

Reply:
Do you think those Swift Guidelines links I mentioned would help here with clarification?